fastlane/Fastfile

```diff
@@ -23,72 +22,73 @@ platform :ios do
-                    contact_first_name: "Christopher",
-                    contact_last_name: "Batin",
-                    demo_account_name: "demo@example.com",
-                    demo_account_password: "N^LhU%h4NV&z",
-                    notes: "App for employees for the company only. We use Google Sign-in as a business solution and therefore are not required to have Sign In with Apple."
-                  },
-                changelog: changelog_from_git_commits)
-        else
-            UI.important "Warning: This action can only be completed on your local machine."
+    lane :update_version_number do
+        version_number_string = UI.input "Please enter the version number (without any letters): "
+        version_number = version_number_string.to_f
+        if version_number == 0.0
+            UI.abort_with_message!("Invalid version number please try again.")
+        end
+        verify = UI.confirm "Confirm? #{version_number_string}"
+        if verify == false
+            UI.abort_with_message!("Incorrect version number, ending lane")
         end
+        increment_version_number(
+            version_number: version_number_string,
+            xcodeproj: iOSProject
+        )
+        increment_build_number(
+            build_number: "#{get_datetime_for_build_version}.#{number_of_commits}",
+            xcodeproj: iOSProject
+        )
+        gradle(
+            project_dir: "./android",
+            properties: {
+                # Version code set in build.gradle not here
+                "versionName" => version_number_string,
+                "prodKeyStorePassword" => ENV["ANDROID_PROD_KEYSTORE_PASSWORD"]
+            }
+          )
+    end
+
+    lane :testflight_distribute do
+        tag = "#{version_number_string}-ios"
+        add_git_tag(tag: tag, force: true)
+        push_git_tags(tag: tag, force: true)
+        match(type: "appstore",
+            skip_google_cloud_account_confirmation: true)
+        gym(workspace: iOSWorkspace,
+            configuration: 'Release-Production',
+            export_method: 'app-store',
+            include_bitcode: true)
+        ENV["GOOGLE_APPLICATION_CREDENTIALS"]="admin-sdk.json"
+        upload_symbols_to_crashlytics(dsym_path: ENV["DSYM_OUTPUT_PATH"],
+            binary_path: './ios/Pods/FirebaseCrashlytics/upload-symbols')
+        upload_to_testflight(
+            demo_account_required: true,
+            api_key_path: './appstore-connect-api-key.json',
+            distribute_external: true,
+            notify_external_testers: true,
+            expire_previous_builds: true,
+            groups: ["Murray App Beta Testers"],
+            beta_app_review_info: {
+                contact_email: "christopher.batin@and.digital",
+                contact_first_name: "Christopher",
+                contact_last_name: "Batin",
+                demo_account_name: "demo@example.com",
+                demo_account_password: "N^LhU%h4NV&z",
```

Comment:
I wonder if this should also be a secret - though I appreciate it wasn't before now.